src/js.c

```diff
@@ -2656,7 +2656,10 @@ _subscribe(natsSubscription **new_sub, jsCtx *js, const char *subject, const cha
             // set only if jsi->consumer is NULL!
             if (jsi->consumer == NULL)
             {
-                DUP_STRING(s, jsi->consumer, info->Name);
+                if (info->Name != NULL)
```

Comment:
But why name would be NULL? It should be set no? I am not sure if just doing that here does not move the problem to some other place.